docs/dsl_inspec.md

```diff
@@ -43,10 +43,14 @@ where
 * `'sshd-8'` is the name of the control
 * `impact`, `title`, and `desc` define metadata that fully describes the importance of the control, its purpose, with a succinct and complete description
 * `desc` when given only one argument it sets the default description. As of InSpec 2.3.4, when given 2 arguments (see: `'rationale'`) it will use the first argument as a header when rendering in Automate
-* `impact` is an float that measures the importance of the compliance results and must be a value between `0.0` and `1.0`. The value ranges are:
-  * `0.0 to <0.4` these are controls with minor criticality
-  * `0.4 to <0.7` these are controls with major criticality
-  * `0.7 to 1.0` these are critical controls
+* `impact` is a string, or float that measures the importance of the compliance results.
+  Valid strings for impact are `none`, `low`, `medium`, `high`, and `critical`. The values are based off CVSS 3.0.
```

Comment:
I'd prefer the word "numeric" (which is widely understood) to "Float" which is a technical word referring to the an internal representation of a value.

Reply:
How about `decimal between 0.0 and 1.0` that measures the ... which is what a float is but more commonly known from math class :)